changes/17362-orbit-and-desktop-version

```diff
@@ -0,0 +1,8 @@
+In GET fleet/hosts/:id response, added the following fields:
+- orbit_version
+  - `orbit_version == null` means this agent is not an orbit agent
+- fleet_desktop_version
+  - `fleet_desktop_version == null` means this agent is not an orbit agent or it is an older version which is not collecting the desktop version
+  - `fleet_desktop_version == ""` means this agent is an orbit agent but does not have fleet desktop
+- scripts_enabled
+  - `scripts_enabled == null` means this agent is not an orbit agent or it is an older version which is not collecting scripts_enabled
```

Comment:
It seems these notes belong to the [REST API docs](https://github.com/fleetdm/fleet/blob/main/docs/REST%20API/rest-api.md) and not on the changelog?

Reply:
Moved to API docs https://github.com/fleetdm/fleet/pull/17813